jax/experimental/jax2tf/examples/tflite/mnist/README.md

```diff
@@ -0,0 +1,92 @@
+# MNIST classification
+
+This directory contains the code necessary to train a simple convolution network on the MNIST dataset, and subsequently convert it using jax2tf into a model that can be converted to tflite format.
+
+This example is based on the [TensorFlow Lite demo of building a handwritten digit classifier app with TensorFlow Lite](https://developer.android.com/codelabs/digit-classifier-tflite). The model code is taken from [FLAX Linen Mnist Example](https://github.com/google/flax/tree/master/linen_examples/mnist).
+
+## Requirements
+
+* TensorFlow dataset mnist will be downloaded and prepared automatically, if necessary
```

Comment:
Suggestion: "preprocessed" (instead of "prepared")

```suggestion
* This example uses [Flax](http://github.com/google/flax) and [TensorFlow Datasets](https://www.tensorflow.org/datasets). The code downloads the MNIST dataset from TensorFlow Datasets and preprocesses it before feeding the data into the neural network as input.
```

LMKWYT @qiuminxu (I'm actually working on a Flax MNIST walkthrough right now 😺)